account-management-api/src/main/java/uk/gov/di/accountmanagement/services/AccountDeletionService.java

```diff
@@ -113,6 +116,10 @@ public DeletedAccountIdentifiers removeAccount(
                                             RequestHeaderHelper.getHeaderValueOrElse(
                                                     n.getHeaders(), SESSION_ID_HEADER, ""))
                             .orElse(AuditService.UNKNOWN);
+            var pairs =
+                    new AuditService.MetadataPair[] {
+                        pair("manualDeletion", manualDeletion.toString())
+                    };
```

Comment:
This variable can probably be inlined as line 134 takes a varargs e.g.

`auditService.submitAuditEvent(DELETE_ACCOUNT, auditContext, pair("manualDeletion", manualDeletion"))`